common/source_model/src/test/scala/weco/catalogue/source_model/sierra/rules/SierraItemAccessTest.scala

```diff
@@ -15,24 +15,33 @@ class SierraItemAccessTest
     extends AnyFunSpec
     with Matchers
     with SierraDataGenerators {
+
+  def createFixedFieldWith(label: String)(value: String,
+                                          display: String = ""): FixedField =
+    FixedField(
+      label = label,
+      value = value,
+      if (display.isEmpty) { Some(display) } else { None }
```

Comment:
Should this b e !display.isEmpty?  IIUIC, this results in either "" or None

Reply:
Yes, it should. Oops.

I'm tempted to omit the `display` field entirely, rather than fix this – it's useful for the readability of the tests and debugging, but the downstream code only uses the single-letter Sierra codes, never the `display` value.